src/services/groups.ts

```diff
@@ -38,16 +38,29 @@ interface GroupResponse {
   isPublic: boolean;
 }
 
-export async function groupsByRegion(region: number): Promise<GroupResponse[]> {
```

Comment:
this function is now basically the same as the one below it, "groups." This is fine I guess but a better solution would be to just include recipients in that query and use that one, rather than proliferating these nearly identical functions

Reply:
Yes you are correct in Garretts re-write he has removed one of these functions. So ultimately the call from AR index will go away.